modules/installation/pages/general-requirements.adoc

```diff
@@ -44,13 +44,13 @@ Depending on your organization's setup, you might also need to activate your sub
 [[install.media]]
 == Obtaining Installation Media
 
-From {slsa}{nbsp}15{nbsp}SP1, {susemgr} Server and Proxy is available as a base product, and can be installed with the {slsa} Unified Installer.
+{susemgr} Server and Proxy can be installed with the {slsa} Unified Installer.
 
 ifeval::[{suma-content} == true]
-Therefore you do not require a separate registration code for {slsa} {sles-version} {sp-version}, only for Server or Proxy
+You only require a valid registration code for {productname}. You do not require a separate code for {slsa} {sles-version} {sp-version}.
 endif::[]
 
-Download {slsa}{nbsp}15{nbsp}SP1 or higher from https://www.suse.com/products/server/download/[SUSE Linux Enterprise Server - Media Download]
+If you already have the installer media for {slsa}{nbsp}15{nbsp}SP1, you can re-use it. Otherwise, download the installer from https://www.suse.com/products/suse-manager/download/[SUSE Manager Media Download]. This page will also allow you to retrieve an evaluation code for {productname} if you don't have a valid registration code yet.
```

Comment:
```suggestion
If you already have installation media for {slsa}{nbsp}15{nbsp}SP1, you can re-use it. 
Otherwise, download the installer from https://www.suse.com/products/suse-manager/download/[SUSE Manager Media Download]. 
This page will also allow you to retrieve an evaluation code for {productname} if you don't have a valid registration code yet.
```